special-pages/pages/history/app/components/App.module.css

```diff
@@ -51,13 +55,53 @@ body {
     padding-right: var(--main-padding-right);
     padding-top: 24px;
 }
+
 .customScroller {
     overflow-y: scroll;
-    scrollbar-gutter: stable;
+    padding-right: calc(var(--main-padding-right) - var(--scrollbar-width));
 }
 
+/**
+ * Windows has access to newer CSS features for styling the scrollbar
+ */
 [data-platform="windows"] {
     .customScroller {
-        padding-right: calc(var(--main-padding-right) - var(--windows-scrollbar));
+        scrollbar-gutter: stable;
+        scrollbar-width: var(--scrollbar-width);
+        scrollbar-color: var(--history-scrollbar-controls-color) var(--history-background-color);
+    }
+}
+
+/**
+ * macOS specific styling to handle dark mode - without this full
+ * override you cannot change the track color :(
+ */
+[data-platform="macos"] {
+    .customScroller {
+        --webkit-thumb-color: rgba(136, 136, 136, 0.8);
+
+        &::-webkit-scrollbar {
+            width: var(--scrollbar-width);
+        }
+
+        &::-webkit-scrollbar-track {
+            background-color: var(--history-background-color)
+        }
+
+        &::-webkit-scrollbar-thumb {
+            background-color: var(--webkit-thumb-color);
+            border-radius: calc(var(--scrollbar-width) / 2);
+
+            /** faking some padding on the thumb */
+            border: 4px solid var(--history-background-color);
+        }
+
+        &::-webkit-scrollbar-button {
+            display: block;
+            background-color: var(--history-background-color);
+
+            /** a small vertical gap to prevent it touching boundaries */
+            height: 2px;
+        }
     }
-}
\ No newline at end of file
+}
```

Comment:
Ideally, we wouldn't need this. But for now it gets the project over the line. The window example above is preferred.